main/templates/main/snippets/home.html

```diff
@@ -0,0 +1,38 @@
+<table>
+    <tr>
+        <td>
```

Comment:
It might be good to use more of a bootstrap layout here e.g.

```suggestion
<div class="container position-relative z-5 pt-0 mt-4 mt-lg-5" data-bs-theme="dark">
    <div class="row justify-content-center text-center pt-md-2 pt-lg-0 pt-xl-2 pb-0 mb-md-2">
        <div class="col-lg-10 col-xl-9 col-xxl-8">
```

Reply:
Absolutely, makes sense. I was just not sure what style to use.